src/components/chat-sidebar/chat-sidebar-link.tsx

```diff
@@ -1,31 +1,36 @@
-import { Star } from "lucide-react";
+import { Trash2 } from "lucide-react";
 import Link from "next/link";
 
 import { Button } from "@/components/ui/button";
 import { SidebarMenuButton, SidebarMenuItem } from "@/components/ui/sidebar";
+import { useChats } from "@/hooks/use-chats";
 
 export function ChatSidebarLink({
-  href,
+  uuid,
   title,
-}: Readonly<{ href: string; title: string }>) {
+}: Readonly<{ uuid: string; title: string }>) {
+  const chats = useChats();
   return (
     <SidebarMenuItem>
       <SidebarMenuButton asChild>
-        <Link href={href}>
+        <Link
+          href={`chats/${uuid}`}
+          className="flex flex-row items-center justify-between"
+        >
           <span className="truncate">{title}</span>
           <Button
             size="icon"
-            className="flex-none"
+            className="flex-none transition hover:text-red-500"
```

Comment:
**praise:** I didn't know about the `truncate` tailwind class, very cool. Nice subtle hover effect too